scriptworker_client/src/scriptworker_client/github_client.py

```diff
@@ -192,7 +192,7 @@ async def get_branch_head_oid(self, branch: str) -> str:
 
         return repo["object"]["oid"]
 
-    async def get_file_listing(self, path: str = "", branch: Optional[str] = None, depth_per_query=10) -> List[str]:
+    async def get_file_listing(self, paths: Union[str, List[str]] = "", branch: Optional[str] = None, depth_per_query=10) -> List[str]:
```

Comment:
I tested this patch in [a real world job](https://treeherder.mozilla.org/jobs?repo=try&revision=7f77bf9dac8e9214be8d113398109f7565960a08&selectedTaskRun=Ql0bArOIS1iLRUXyGy03Kw.0). It ended up failing due to the graphql query timing out. Moving the `depth_per_query` down to `5` fixed that. Let's make that the new default, since this function will often do more work now.

Reply:
cool :fast_forward: